pages/landing-page.js

```diff
@@ -0,0 +1,437 @@
+                  {'Give Your Time'}
+                </a>
+              </Link>
+              <Link href="/partner-with-us">
+                <a className="btn btn-ghost btn-sm rounded-btn">
+                  {'Partner With Us'}
+                </a>
+              </Link>
+            </div> */}
+          </div>
+          <div className="pr-7 hidden">{' Help Families in Need'}</div>
+          <div className="flex-row hidden sm:flex">
+            <button className="btn btn-square btn-ghost pr-6" type="button">
+              <FaRegHeart className="inline-block w-5 h-5 stroke-current" />
+            </button>
+          </div>
+
+          {/* <div className="flex-none hidden sm:flex">
+            <button className="btn btn-square btn-ghost" type="button">
+              <BsSearch className="inline-block w-5 h-5 stroke-current" />
+            </button>
+          </div> */}
+        </div>
+      </div>
+      <style jsx>{`
+        .hidden {
+          display: block;
+        }
+        .nav-bar {
+          opacity: ${navOpacity};
+          background-color: #2159633d !important;
+          box-shadow: 0 0.3px 7.2px 0px rgba(0, 0, 0, 0.12);
+          z-index: 1;
+          position: fixed;
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          top: ${navOpacity ? navOpacity - 1 : -3}rem;
+          left: 0;
+          right: 0;
+          height: 73px;
+          transition: all ${fadeInSpeed} ease-in-out;
+        }
+        @media (max-width: 768px) {
+          .nav-bar {
+            top: 0;
+          }
+          .hidden {
+            display: none;
+          }
+        }
+      `}</style>
+    </>
+  );
+};
+
+// -----------------------------------------------------------------------------------------------------------------------
+// Export LandingPageNav end.
+// -----------------------------------------------------------------------------------------------------------------------
+
+// -----------------------------------------------------------------------------------------------------------------------
+// TODO: Export TestimonialSection to components folder or leave it here as it's not the most reusable component.
+// -----------------------------------------------------------------------------------------------------------------------
+
+const TestimonialSection = () => {
+  return (
+    <section className="testimonial-section">
+      <div className="section-title">
+        <h2>{'See What People Are Saying'}</h2>
+      </div>
+      <div className="cards">
+        <div className="testimonial-card">
+          <div className="card glass lg:card text-neutral-content">
+            <figure className="p-0">
+              <Image
+                alt="testimonial"
+                className="shadow-lg"
+                height={340}
+                layout="responsive"
+                src="/images/HealthWorkShop.jpg"
+                width={500}
+              />
```

Comment:
A quick pointer regarding images, it's ideal to import the image as it automatically provides the correct width and height, ensuring the image looks as good as possible.

```js
import HealthWorkshopImage from '/images/HealthWorkshop.jpg';
...
src={HealthWorkshopImage}
```